specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -440,6 +440,99 @@
         }
       }
     },
```

Comment:
Highly recommend putting these new APIs in a preview version for initial release

Reply:
I moved the code to preview